hbase/src/test/java/akka/stream/alpakka/hbase/javadsl/HBaseStageTest.java

```diff
@@ -65,10 +65,35 @@ public static void teardown() {
         } catch (UnsupportedEncodingException e) {
           e.printStackTrace();
         }
-        return put;
+        return Arrays.asList(put);
       };
   // #create-converter
 
+  // #create-converter-mutations
+  Function<Person, List<Mutation>> mutationsHBaseConverter =
+      person -> {
+        try {
+          Put put = new Put(String.format("id_%d", person.id).getBytes("UTF-8"));
+          put.addColumn(
+              "info".getBytes("UTF-8"), "name".getBytes("UTF-8"), person.name.getBytes("UTF-8"));
+
+          Append append = new Append(String.format("id_%d", person.id).getBytes("UTF-8"));
+          append.add(
+              "info".getBytes("UTF-8"), "aliases".getBytes("UTF-8"), person.name.getBytes("UTF-8"));
+
+          Delete delete = new Delete(String.format("id_%d", person.id).getBytes("UTF-8"));
+
+          Increment increment = new Increment(String.format("id_%d", person.id).getBytes("UTF-8"));
+          increment.addColumn("info".getBytes("UTF-8"), "age".getBytes("UTF-8"), 1);
+
+          return Arrays.asList(append, put, delete, increment);
```

Comment:
This example does not make much sense, could you come up with something more realistic?

Reply:
In my real use case I just need to `Put` or `Delete` depending of some message data.

For the pull request, now I have tried to split every mutations in a separate example. Probably a more realistic scenario require a more complex data model. 

About the `Append` I'm not sure how to use it. If I understand correctly it will append bytes to a single cell, I can't see a real usage for now (maybe for a log appender?)...

In any case I think that the point of this pull requests is to permit sending different kind of mutations. Explaining how to use HBase mutations is probably out of scope in my opinion. Eventually we can link some official HBase doc?

thanks